docs/getting-started/deployment/infrastructure-as-code/overview.md

```diff
@@ -0,0 +1,21 @@
+# Infrastructure as Code
+
+CarbyneStack has adopted Infrastructure as Code (IaC) as a core principle. IaC is the process of managing and provisioning infrastructure through code instead of manually deploying resources via kubectl, helm, etc. This allows for the infrastructure to be versioned, tested, and deployed in a repeatable manner.
```

Comment:
It's `Carbyne Stack` not `CarbyneStack`; check whole PR please.

Reply:
fixed